ui/app/AppLayouts/Chat/views/ChatMessagesView.qml

```diff
@@ -50,7 +50,7 @@ Item {
         readonly property bool isMostRecentMessageInViewport: chatLogView.visibleArea.yPosition >= 0.999 - chatLogView.visibleArea.heightRatio
         readonly property var chatDetails: chatContentModule.chatDetails || null
 
-        readonly property var loadMoreMessagesIfScrollBelowThreshold: Backpressure.oneInTime(root, 500, function() {
+        readonly property var loadMoreMessagesIfScrollBelowThreshold: Backpressure.oneInTime(root, 100, function() {
```

Comment:
@osmaczko how do you feel about reducing the threshold to 100?
Having it to 500 will reduce the feeling of "infinite scroll" and the user needs to often press Fetch more messages button to fetch more messages.

I've done some tests on the #test channel to scroll and load all the messages and these are the finings:
```
500 threshold:
29 nim requests
518 loadMoreMessagesIfScrollBelowThreshold calls with d.scrollY < 1000
9 Hits on Fetch more messages
```
```
100 threshold:
29 nim requests
202 loadMoreMessagesIfScrollBelowThreshold calls with d.scrollY < 1000
0 Hits on Fetch more messages
```